ActivityHistoryApi/V1/Infrastructure/ActivityHistoryDB.cs

```diff
@@ -0,0 +1,40 @@
+using ActivityHistoryApi.V1.Domain;
+using Amazon.DynamoDBv2.DataModel;
+using Hackney.Core.DynamoDb.Converters;
+using System;
+using Type = ActivityHistoryApi.V1.Domain.Type;
+
+namespace ActivityHistoryApi.V1.Infrastructure
+{
+    [DynamoDBTable("ActivityHistory", LowerCamelCaseProperties = true)]
+    public class ActivityHistoryDB
+    {
+        [DynamoDBRangeKey]
+        public Guid Id { get; set; }
+
+        [DynamoDBHashKey]
+        public Guid TargetId { get; set; }
+
+
+        [DynamoDBProperty(Converter = typeof(DynamoDbEnumConverter<Type>))]
+        public Type Type { get; set; }
+
+        [DynamoDBProperty(Converter = typeof(DynamoDbEnumConverter<TargetType>))]
+        public TargetType TargetType { get; set; }
+
+        [DynamoDBProperty(Converter = typeof(DynamoDbDateTimeConverter))]
+        public DateTime CreatedAt { get; set; }
+
+        public int TimetoLiveForRecordInDays { get; set; }
```

Comment:
You should check what this is used for...
If this means that after this many days the record "dies", then we should verify what is meant by that. It could be:
- Soft delete - i.e. a flag is set on the object to say it is now "deleted". This would mean altering any query to filter out "deleted" ones , and having some sort of back end thread that polled periodically (daily?) to update any records that should now be "deleted"
- Hard delete - in this case we can alter the DynamoDb table configuration to get AWS to remove the record for us.

Reply:
I need to change the name to `timeToLiveForRecord `, Mehdi told me that it is an optional field for automatic purge made by DynamoDB. It is an Epoch timestamp, So I don't think we will need to a delete

Reply:
Ok... but then we will need to change the Table definition in terraform to specify which field is the TTL one.
And we should also explain in swagger exactly how this value should be populated. See [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-before-you-start.html).

Reply:
I've updated the terraform, I've added an example to swagger, the value should be populated as an Epoch timestamp, for example, 1645119622